README.md

```diff
@@ -3,10 +3,8 @@
 Solved problems of HackerRank.
 www.hackerrank.com
 
-All solutions were developed by Daniel Henrique (me).
-https://www.hackerrank.com/daniel_henrique1
-
-Some solutions are very old and not reflect my current skills.
+The problems are sorted according to the HackerRank Hierarchy.<br>
+Please contribute if you have a better solution or add a new solution!
```

Comment:
That PR (https://github.com/danhenriquesc/hackerrank/pull/26) was merged with this solution, but I like the part of 
```
The problems are sorted according to the HackerRank Hierarchy.<br>
Please contribute if you have a better solution or add a new solution!
```

Please, pull the last commits of master, add this snippet and change
```
Solved problems of HackerRank. www.hackerrank.com
```

To something better, that had the link too (`Solved problems of HackerRank.` currently is ambiguous with `
A collection of solutions and explanations of Hackerrank.com problems in many languages.`)